wrappers/python/readme.md

```diff
@@ -53,7 +53,7 @@ For example: `-DPYTHON_EXECUTABLE=C:/Python27/python.exe`
 
 4. Open `librealsense2.sln` that was created in the previous step, and build the `pyrealsense2` project
 5. Open the output folder of the project (e.g `build\x64-Release\Release\`) and copy `pyrealsense2.pyd` into your python's `site-packages` (e.g `C:\Python27\Lib\site-packages`)
-6. Alternatively, copy the build output (`realsense2.dll` and `pyrealsense2.pyd`) next to your script.
+6. Alternatively, copy the build output (`librealsense2.dll` and `pyrealsense2.pyd`) next to your script.
```

Comment:
On windows, the dll is named `realsense2.dll`, no `lib` prefix is appended...

Reply:
Fix the lib name for the dll